tests/integration/test_pip.py

```diff
@@ -89,3 +91,88 @@ def test_pip_packages(
     _ = utils.fetch_deps_and_check_output(
         tmpdir, test_case, test_params, source_folder, test_data_dir, cachi2_image
     )
+
+
+
+@pytest.mark.parametrize(
+    "test_params,check_cmd",
+    [
+        # Test case checks fetching pip dependencies, generating environment vars file,
+        # building image with all prepared prerequisites and testing if pip packages are present
+        # in built image
+        pytest.param(
+            utils.TestParameters(
+                repo="https://gitlab.cee.redhat.com/osbs/osbs-test-cachito-project.git",
```

Comment:
I know that this is still a draft, but mentioning this just in case: we can't use a repo hosted in gitlab.cee.redhat.com for upstream tests